{
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0",
    "cveMetadata": {
        "cveId": "CVE-2024-22309",
        "assignerOrgId": "21595511-bba5-4825-b968-b78d1f9984a3",
        "state": "PUBLISHED",
        "assignerShortName": "Patchstack",
        "dateReserved": "2024-01-08T20:58:59.274Z",
        "datePublished": "2024-01-24T11:42:42.859Z",
        "dateUpdated": "2024-01-24T11:42:42.859Z"
    },
    "containers": {
        "cna": {
            "affected": [
                {
                    "collectionURL": "https://wordpress.org/plugins",
                    "defaultStatus": "unaffected",
                    "packageName": "chatbot",
                    "product": "ChatBot with AI",
                    "vendor": "QuantumCloud",
                    "versions": [
                        {
                            "changes": [
                                {
                                    "at": "5.1.1",
                                    "status": "unaffected"
                                }
                            ],
                            "lessThanOrEqual": "5.1.0",
                            "status": "affected",
                            "version": "n/a",
                            "versionType": "custom"
                        }
                    ]
                }
            ],
            "credits": [
                {
                    "lang": "en",
                    "type": "finder",
                    "user": "00000000-0000-4000-9000-000000000000",
                    "value": "Le Ngoc Anh (Patchstack Alliance)"
                }
            ],
            "descriptions": [
                {
                    "lang": "en",
                    "supportingMedia": [
                        {
                            "base64": false,
                            "type": "text/html",
                            "value": "Deserialization of Untrusted Data vulnerability in QuantumCloud ChatBot with AI.<p>This issue affects ChatBot with AI: from n/a through 5.1.0.</p>"
                        }
                    ],
                    "value": "Deserialization of Untrusted Data vulnerability in QuantumCloud ChatBot with AI.This issue affects ChatBot with AI: from n/a through 5.1.0.\n\n"
                }
            ],
            "metrics": [
                {
                    "cvssV3_1": {
                        "attackComplexity": "HIGH",
                        "attackVector": "NETWORK",
                        "availabilityImpact": "NONE",
                        "baseScore": 8.7,
                        "baseSeverity": "HIGH",
                        "confidentialityImpact": "HIGH",
                        "integrityImpact": "HIGH",
                        "privilegesRequired": "NONE",
                        "scope": "CHANGED",
                        "userInteraction": "NONE",
                        "vectorString": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:C/C:H/I:H/A:N",
                        "version": "3.1"
                    },
                    "format": "CVSS",
                    "scenarios": [
                        {
                            "lang": "en",
                            "value": "GENERAL"
                        }
                    ]
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "cweId": "CWE-502",
                            "description": "CWE-502 Deserialization of Untrusted Data",
                            "lang": "en",
                            "type": "CWE"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "orgId": "21595511-bba5-4825-b968-b78d1f9984a3",
                "shortName": "Patchstack",
                "dateUpdated": "2024-01-24T11:42:42.859Z"
            },
            "references": [
                {
                    "tags": [
                        "vdb-entry"
                    ],
                    "url": "https://patchstack.com/database/vulnerability/chatbot/wordpress-ai-chatbot-plugin-5-1-0-unauthenticated-php-object-injection-vulnerability?_s_id=cve"
                }
            ],
            "solutions": [
                {
                    "lang": "en",
                    "supportingMedia": [
                        {
                            "base64": false,
                            "type": "text/html",
                            "value": "Update to&nbsp;5.1.1 or a higher version."
                        }
                    ],
                    "value": "Update toÂ 5.1.1 or a higher version."
                }
            ],
            "source": {
                "discovery": "EXTERNAL"
            },
            "title": "WordPress ChatBot Plugin <= 5.1.0 is vulnerable to PHP Object Injection",
            "x_generator": {
                "engine": "Vulnogram 0.1.0-dev"
            }
        }
    }
}